javascript/javascript1/week3/speed.js

```diff
@@ -0,0 +1,12 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  function notThisFunctionName(travelInformation){
```

Comment:
You should rename this function to something more meaningful. One helpful way is to think in terms of what the function is doing.